apps/settings/js/activity_handler.js

```diff
@@ -77,13 +77,6 @@
             document.body.dataset.filterBy =
               activitySource.data.filterBy || 'all';
             break;
-          case 'call':
```

Comment:
The pendingTargetPanel name will be checked again in Main.js, so we can safely remove the check from here
https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/js/main.js#L58